acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
```

Comment:
Side note unrelated to this PR, it seems like if we add / start testing against server 2012 this test will start failing